[2021-10-09 00:00:01,096] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-08T00:00:00+00:00 [queued]>
[2021-10-09 00:00:01,126] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-08T00:00:00+00:00 [queued]>
[2021-10-09 00:00:01,128] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-09 00:00:01,129] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-09 00:00:01,130] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-09 00:00:01,141] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_B> on 2021-10-08T00:00:00+00:00
[2021-10-09 00:00:01,147] {standard_task_runner.py:52} INFO - Started process 56718 to run task
[2021-10-09 00:00:01,163] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_B', '2021-10-08T00:00:00+00:00', '--job-id', '165', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmptxf3v5pw', '--error-file', '/tmp/tmp_8r_8_fl']
[2021-10-09 00:00:01,166] {standard_task_runner.py:77} INFO - Job 165: Subtask Training_Model_B
[2021-10-09 00:00:01,252] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_B 2021-10-08T00:00:00+00:00 [running]> on host d6b4131456ad
[2021-10-09 00:00:01,353] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_B
AIRFLOW_CTX_EXECUTION_DATE=2021-10-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-08T00:00:00+00:00
[2021-10-09 00:00:01,400] {python.py:151} INFO - Done. Returned value was: None
[2021-10-09 00:00:01,419] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_B, execution_date=20211008T000000, start_date=20211009T000001, end_date=20211009T000001
[2021-10-09 00:00:01,495] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-09 00:00:01,553] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
